com.cyberbotics.Webots.yaml

```diff
@@ -0,0 +1,53 @@
+app-id: com.cyberbotics.Webots
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: webots
+separate-locales: false
+base: io.qt.qtwebengine.BaseApp
+base-version: '6.5'
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Notifications
+  - --device=all
+  - --allow=devel
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - shared-modules/glu/glu-9.json
+  - name: Webots
+    buildsystem: simple
+    build-commands:
+      - install -d -m755 /app/webots_dir/
+      - tar -xjf webots.tar.bz2 -C /app/webots_dir/ --strip-components=1
+      - install -D -m755 webots ${FLATPAK_DEST}/bin/webots
+      - install -D -m644 /app/webots_dir/resources/icons/core/webots.png ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.png
+      - install -D -m644 -t ${FLATPAK_DEST}/share/applications/ ${FLATPAK_ID}.desktop
+      - install -D -m644 -t ${FLATPAK_DEST}/share/metainfo/ ${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: file
+        url: https://github.com/cyberbotics/webots/releases/download/R2023b/webots-R2023b-x86-64.tar.bz2
```

Comment:
the description say it's crossplatform but you limit the architecture to PC.

Why not building from source instead?